packages/icons/src/svg/launch.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
per our previous dicussion all element of an icon should be wrapped into a root <g/> element so filters could be applied on IE11 to.